### ISSUE TITLE
error parsing configuration file in gnatcoll-traces

### ISSUE THREAD

**Issue by eqcola** [AdaCore/gnatcoll-core#35]
Hi,
in older version i've configuration file that contains row like:

```--Non cancellate questo file, serve ad attivare i debug nel framework ADA GNAT ARI
+
>>log
PLCTIME=yes >> logPLCTIME
PLCCOMM=yes >> logPLCCOMM
DEBUG.COLORS=no
DEBUG.ABSOLUTE_TIME=yes
DEBUG.STACK_TRACE=no
DEBUG.COUNT=no
DEBUG.ENCLOSING_ENTITY=no
DEBUG.LOCATION=yes
```

the new version raise :
`
raise Constraint_Error with
                 "Line " & Count'Img & ": """ & Line & """ is not recognised.";
`
in attempt to space between >> and stream.
`>> logPLCTIME` 
Thanks,
regards.

**Comment by t-14** [comment]
Thank you for your report (note, I've edited it to fix formatting). Indeed, looks like a regression, we will investigate.

**Comment by eqcola** [comment]
> Thank you for your report (note, I've edited it to fix formatting). Indeed, looks like a regression, we will investigate.

ok thanks

**Comment by t-14** [comment]
This is fixed by c6b32eb
